rack-protection/lib/rack/protection/authenticity_token.rb

```diff
@@ -9,14 +9,78 @@ module Protection
     # Supported browsers:: all
     # More infos::         http://en.wikipedia.org/wiki/Cross-site_request_forgery
     #
-    # Only accepts unsafe HTTP requests if a given access token matches the token
-    # included in the session.
+    # This middleware only accepts requests other than <tt>GET</tt>,
+    # <tt>HEAD</tt>, <tt>OPTIONS</tt>, <tt>TRACE</tt> if their given access
+    # token matches the token included in the session.
```

Comment:
Sorry, my last comment here was vague. I meant to just tweak the comment this way:

This middleware only accepts unsafe HTTP requests (requests other than <tt>GET</tt>, <tt>HEAD</tt>, <tt>OPTIONS</tt>, <tt>TRACE</tt>) if their given access token matches the token included in the session.